### PR TITLE
Fill MsgBase ALL

### PR DESCRIPTION
**Pull request by smellthemoon**
Signed-off-by: xinguo.li <xinguo.li@zilliz.com>
#19639

### CODE REVIEW

**Comment by smellthemoon** [comment]
rerun ut

File: internal/util/commonpbutil/commonpbutil.go [hunk 1]
```diff
@@ -57,20 +55,21 @@ func GetNowTimestamp() uint64 {
 	return uint64(time.Now().Unix())
 }
 
-func SetTargetID(msgBase *commonpb.MsgBase, targetID int64) *commonpb.MsgBase {
+func UpdateMsgBase(msgBase *commonpb.MsgBase, targetID int64) *commonpb.MsgBase {
```
Comment (jiaoew1991) [inline internal/util/commonpbutil/commonpbutil.go:58]: 1. Rename to FillXXX is better
2. change the param from targetID to MsgBasOptions is better

Comment (jiaoew1991) [reply internal/util/commonpbutil/commonpbutil.go:58]: maybe combine this one with NewMsgBaseCopy is better 🤔

**Pull request merged**